.gitignore

```diff
@@ -1,12 +0,0 @@
-.*
```

Comment:
Why delete this file?

Reply:
Not intended to. Don't know why there's a ".*" in .gitignore, which messed up these two files. Removing this line.